mlflow/store/tracking/dbmodels/models.py

```diff
@@ -555,7 +555,11 @@ class SqlInput(Base):
     __tablename__ = "inputs"
     __table_args__ = (
         PrimaryKeyConstraint(
-            "source_type", "source_id", "destination_type", "destination_id", name="inputs_pk"
+            "source_type",
+            "source_id",
+            "destination_type",
+            "destination_id",
+            name="inputs_pk",
```

Comment:
sorry I'm not sure which part of VSCode's autoformatting did this (probably black)

pre-commit ruff didn't revert this so I'm assuming this is ok?